frontend/src/components/EditableName.vue

```diff
@@ -30,26 +31,38 @@ onUpdated(() => {
 </script>
 
 <template>
-  <span :class="{ 'd-none': !editing }">
+  <template v-if="props.noToggle === true">
```

Comment:
I don't think it's worthwhile to regroup both toggle and noToggle modes in the same component, since they are displaying very different things and no code is actually shared between the two modes, is it?
I would just put the `FormKit` element in the `NewRuleForm`, since it's the only place where it's called with `noToggle`.

Reply:
you are right -- i was def ovethinking this one a bit :)

have updated it now to just use a text input vias formkit